lib/rules/require-ember-lifeline.js

```diff
@@ -0,0 +1,48 @@
+/**
+ * @fileOverview Require the use of ember-lifeline's runTask over using Ember.run.later.
+ * @author Steve Calvert
```

Comment:
After spending a bit of time thinking about it, I'd like for us to stop placing `@author` annotations in these files. They feel territorial and might off put newer contributors.

If we want to acknowledge authors/contributors, we should go the route of placing an `AUTHORS.txt` file in the root of the repo which can be generated from the git history ([example from QUnit](https://github.com/qunitjs/qunit/blob/master/AUTHORS.txt)).